vsts/vsts.yaml

```diff
@@ -146,14 +146,127 @@ jobs:
             performMultiLevelLookup: true
             installationPath: $(Agent.ToolsDirectory)/dotnet
 
+      - task: Docker@2
```

Comment:
TODO: consolidate these on a single template and not duplicated script in two areas.